src/modules/ipc.cpp

```diff
@@ -123,8 +123,14 @@ namespace modules {
       }
 
       broadcast();
+      return;
```

Comment:
```suggestion
```

Reply:
This could be left as an optimization? Once it matches the first hook, it cannot match the others.